assignment1/numpy.ipynb

```diff
@@ -867,10 +867,11 @@
    "outputs": [],
    "source": [
     "from scipy import misc\n",
+    "import skimage\n",
     "import matplotlib.pyplot as plt\n",
     "\n",
     "img = misc.face()\n",
-    "img = misc.imresize(img, (720, 960))\n",
+    "img = skimage.transform.resize(img, (720, 960))\n",
```

Comment:
@mrshu I would change the import to `from skimage import transform` and then use just
```suggestion
    "img = transform.resize(img, (720, 960))\n",
```
as this would be more consistent with how other function from the specific parts of the packages are used (the package name is omitted), but this is just a nitpick. Otherwise, LGTM and can be merged.